src/components/Redemptions/RedemptionsPage.tsx

```diff
@@ -154,15 +190,15 @@ class RedemptionsPage extends React.Component<IProps, null> {
   }
 
   private renderTotalRewards(): RenderOutput {
-    const { currentAccountAddress, data: [, proposals] } = this.props;
+    const { currentAccountAddress, data: proposals } = this.props;
 
     const genReward = new BN(0);
     const ethReward = new BN(0);
     const externalTokenRewards: { [symbol: string]: BN } = {};
     const reputationRewardDaos = new Set();
 
     // Calculate the total rewards from the genesisprotocol
-    proposals.forEach((proposal) => {
+    proposals.forEach((proposal: any) => {
```

Comment:
can we not use `any` here?  Is there not a common interface?

Reply:
this is arbitrary data from a special query, so no we cant

Reply:
The `proposal` object implements properties and methods that we are using.  We can certainly define the interface that `proposal` is expected to implement.